hs-bindgen/test/internal/test-internal.hs

```diff
@@ -2,22 +2,23 @@
 module Main (main) where
 
 import Control.DeepSeq (force)
-import Control.Exception (try, displayException, evaluate)
+import Control.Exception (displayException, evaluate, try)
```

Comment:
is it your editor adding those changes?

Please don't. These are  (mostly) noise: they are not related to the change at hand.

If you really must, please put those into separate commits.

Reply:
Yes, I have configured my editor to auto-format imports which saves a lot of time. I configured the formatting so that it it honors the current way we format our imports. It does sort htings alphabetically though, and I think this is a good thing.

If you insist, I can put changes like these into separate commits.

Reply:
Actually, I thought about this a bit more, and it is not too much of a hazzle for me to create separate commits if I know this from the beginning. In this PR, the changes are interleaved into the different commits, I only see one easy way out: Squash all commits in this PR into one, and extract the commits ordering existing imports. Would that be OK for you, @phadej ?

Reply:
I extracted the relevant changes into a separate commit. My opinion though is that such changes should be seen as (girls scout) improvements, and not as changes with merge-conflict potential. I hope the status quo is OK with you!

Reply:
TBH, I'm usually more annoyed with such "formatting" changes. If people have editors configured differently, those changes go back and forth, so I'm basically against them unless they are enforced.

We could use `stylish-haskell` and **enforce** its usage in CI, then there won't be style pendulums (and FWIW there are, people format module export lists, and even in this project they have been formatted forward and back).

This is annoying issue.

Reply:
I understand your point. What about enforcing them then? I have a working `stylish-haskell` setup that **only sorts and formats imports**, and does this in an unintrusive way. In particular, it keeps import groups when they are separated by a newline. `stylish-haskell` comes with HLS, so we can assume everybody has it installed. I could add the check to CI and commit my configuration. What do you think? @edsko @sheaf @TravisCardwell @phadej

Reply:
I had a bad experience with formatters with the Cabal project. The main issue I had was that the formatter would increase the size of diffs because one small change could lead the formatter to decide a whole block of code needed different indentation. This then caused major headaches when rebasing. I don't think this is a concern in this case if it's just about imports/exports.

I also would want to make sure that formatting failures don't block or cancel other CI jobs, as it is a very frustrating workflow when one wants CI results.